api_v2/management/commands/import.py

```diff
@@ -28,5 +31,151 @@ def handle(self, *args, **options) -> None:
             exit(0)
 
         fixture_filepaths = glob.glob(options['dir'] + '/**/*.json', recursive=True)
+        
+        self.stdout.write(f'Found {len(fixture_filepaths)} fixture files to load.')
```

Comment:
This section modifies the import flow in the following ways:

- Integrity checks are held off while fixtures are loading, to allow circular dependencies to succeed. Now necessary due to new relationships between SRD 2014 and 2024
- After all data is loaded, integrity checks are run
- All errors (not just one) are printed to the terminal in case of failure
- This now includes both the item and nature of the error for each issue, to speed up debugging.